docs/architecture/ADR-002-QGB-Overview.md

```diff
@@ -0,0 +1,47 @@
+# ADR 003: ValSet
+
+## Changelog
+
+- {date}: {changelog}
+
+## Context
+Various Rollups on EVM L1s may have a need for using Celestia as a DA layer, while keeping fraud/validity proof verification in the EVM.
+To that end, two concrete possibilities exist: the Gravity bridge and a native EVM light client.
```

Comment:
as discussed in the sync, we want to focus on the app integration portion here. I'm still thinking on exactly what needs to be covered, because technically we don't need any of the app integration until slashing. If we didn't need to slash validators, then we would do everything off chain to save state bloat.